### ISSUE TITLE
Refactor how API/file content is served

### ISSUE THREAD

**Issue by SpencerDawson** [ecksbee/kushim#9]
The `/networks` call pulls up a list of concept networks for any given taxonomy set. We want to restructure so instead of "API endpoints", we are serving the files that are generated for consumption.

This relates to #8.

**Comment by joshuanario** [comment]
Just one taxonomy set at a time.

**Comment by joshuanario** [comment]
Consider restructring `gts` directory as follows:

```
/gts
   /concepts ##schema and linkbase files
   /taxonomies
      /_.json ## maps a guid to a taxonomy code { b488c7ef-4a6e-4e98-a78c-14b32e9afc5f: usgaap2024 }
      /{guid} ##folder for serialized concept cards and renderables
           /_.json ##metadata on installed taxonomy set

**Comment by joshuanario** [comment]
Closed by #13
